enterprise/api/v1/mixins.py

```diff
@@ -84,7 +91,10 @@ def update_course_runs(self, course_runs, enterprise_customer, enterprise_contex
                 course_run=course_run,
                 query_parameters=enterprise_context,
             )
-            enrollment_url = enterprise_customer.get_course_run_enrollment_url(course_run.get('key'))
+            # Update querystring of enterprise course enrollment url with utm context.
```

Comment:
Here we can use the same approach as used for updating `marketing_url `

```
enrollment_url = enterprise_customer.get_course_run_enrollment_url(course_run.get('key'))
course_run.update({'enrollment_url': utils.update_query_parameters(marketing_url, enterprise_context)})
```

Reply:
I do not think updating query parameters of `enrollment_url ` using `enterprise_context ` would cause unwanted results but if they do then I think you can use `enterprise_context` to create another dict that only contains the desired key value pairs in this method.